src/current/_includes/molt/replicator-flags.md

```diff
@@ -66,21 +66,21 @@ The following flags are set with [`--replicator-flags`](#global-flags) and can b
-| `--tlsPrivateKey`           | `STRING`   | A path to a PEM-encoded TLS private key.                                                                                                                                        |
-| `--tlsSelfSigned`           |            | If true, generate a self-signed TLS certificate valid for `localhost`.                                                                                                          |
\ No newline at end of file
+|            Flag            |    Type    |                                                                                                                              Description                                                                                                                               |
+|----------------------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--assumeIdempotent`       |            | Disable the extra staging table queries that debounce non-idempotent redelivery in changefeeds.                                                                                                                                                                        |
+| `--bestEffortOnly`         |            | Eventually-consistent mode; useful for high-throughput, skew-tolerant schemas with [foreign keys]({% link {{ site.current_cloud_version }}/foreign-key.md %}).                                                                                                         |
+| `--bestEffortWindow`       | `DURATION` | Use an eventually-consistent mode for initial backfill or when replication is behind; `0` to disable.<br><br>**Default:** `1h0m0s`                                                                                                                                     |
+| `--bindAddr`               | `STRING`   | The network address to bind to.<br><br>**Default:** `":26258"`                                                                                                                                                                                                         |
+| `--disableAuthentication`  |            | Disable authentication of incoming Replicator requests; not recommended for production.                                                                                                                                                                                |
+| `--enableCheckpointStream` |            | Enable checkpoint streaming (use an internal changefeed from the staging table for real-time updates)), rather than checkpoint polling (query the staging table for periodic updates), for failback replication. <br><br>**Default:** `false` (use checkpoint polling) |
```

Comment:
`(use an internal changefeed from the staging table for real-time updates))`

Extra end parenthesis